index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>El amigo</title>
+</head>
+<body>
+    <main>
+        <header>
+            <img src="./images/logo.png" alt="Logo">
+        </header>
+    <section>
+    <div class="box1">
+        <h1>The Best Craft Beer in Spain!</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aspernatur neque vero sunt aliquid officia pariatur qui. Accusantium nostrum pariatur corrupti.</p>
+    </div>
+    <div class="box2">
+        <h2>Make a reservation</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. A, placeat.</p>
+        <form action="#" method="GET" id="form">
+            <label for="name" class="label">Your Name</label>
+            <input type="text" name="name" class="input" placeholder="Maxi Mustermaus">
```

Comment:
beim input fehlt das id. dieses verbindet label mit input für screenreader usw.
https://github.com/D02-1/help-and-reference/blob/main/2_uib/4_data/input.md